src/main/java/org/usfirst/frc/team1294/robot/commands/ClimbRope.java

```diff
@@ -0,0 +1,37 @@
+package org.usfirst.frc.team1294.robot.commands;
+
+import org.usfirst.frc.team1294.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class ClimbRope extends Command {
+
+  public ClimbRope() {
+    super("Drive motor");
+    requires(Robot.climbingSubsystem);
+  }
+
+  @Override
+  protected void initialize() {
+
+  }
+
+  @Override
+  protected void execute() {
+    Robot.climbingSubsystem.setMotor(-0.75);
```

Comment:
Did someone on the climbing team specify a slower speed? Or is this just a guess of ours?